### ISSUE TITLE
[rc1] Route not rendered when directly requested

### ISSUE THREAD

**Issue by gelidus** [angular/angular#8545]
Using the new `@Routes()` decorator won't render the component in the `router-outlet` when a page is refreshed on that link. I want to have the default route which will render the `LandingComponent`, but instead I get a blank page. Same if I request directly `/platform`. Both components have a simple hello world in them and when redirected by `[routerLink]`, everything works fine.

Can this be caused by the lazy loader? If yes, how can i temporary disable it?

**Steps to reproduce and a minimal demo of the problem**
- create new app using angular-cli
- add 2 routes (landing and platform)
- request one of the routes

```ts
import { Component } from '@angular/core';
import { LandingComponent } from './+landing';
import { Routes , ROUTER_DIRECTIVES} from '@angular/router';
import { PlatformComponent } from './+platform';

@Component({
  moduleId: module.id,
  selector: 'my-app',
  templateUrl: 'my.component.html',
  styleUrls: ['my.component.css'],
  directives: [ROUTER_DIRECTIVES]
})
@Routes([
  {path: '/', component: LandingComponent},
  {path: 'platform', component: PlatformComponent}
])
export class MyAppComponent {
}
```

**Current behavior**
When the route is directly requested, a blank page is rendered instead of the component that belongs to the path.

**Expected/desired behavior**
Router should be able to automatically get the component for the route and render it.

**Comment by zoechi** [comment]
I guess you need to configure the server to support HTML5 pushState or the client to use `HashLocationStrategy` http://stackoverflow.com/questions/31415052/angular-2-0-router-not-working-on-reloading-the-browser

**Comment by dominique-mueller** [comment]
Maybe related to #8481 ?

**Comment by gelidus** [comment]
@dominique-mueller yes, I think this is duplicate now. Just realized if I add the routerLink somewhere in the app, everything works properly. Should I close?

**Comment by brandonroberts** [comment]
@gelidus yes, please close this one as a duplicate.

**Comment by gelidus** [comment]
Closed as a duplicate #8481
